layouts/hub.tsx

```diff
@@ -42,9 +42,8 @@ const HubLayout = ({ children }: { children: React.ReactNode }) => {
       }
     }
 
-    getUser()
-      .catch(console.error)
-      .then(() => {});
+    // eslint-disable-next-line no-console
+    getUser().catch(console.error);
```

Comment:
ESLint was erroring becaused of this. Not sure how it snuck in in previous merges.